store/sqlstore/init_test.go

```diff
@@ -0,0 +1,9 @@
+package sqlstore
```

Comment:
I can't remember the context here, but I'm guessing it's non-trivial to just push `initStores` and `tearDownStores` into `sqlstore_test` altogether?

Reply:
Actually `store/sqlstore` is being imported in `testlib.helper` and `testlib.helper` is being imported in `store/sqlstore_test`, so importing `store/sqlstore` into `store/sqlstore_test` would cause circular dependency.
If you mean moving those funtions to `store/sqlstore_test` altogether then there would still be issue of circular dependency since `initStores` is using `store/storetest` and same is being imported by `testlib.helper`.

Reply:
Ah, right. Fine as-is. Would love to decouple the sqlstore in the future to allow more flexibility.